pom.xml

```diff
@@ -22,6 +22,11 @@
 				<type>pom</type>
 				<scope>import</scope>
 			</dependency>
+			<dependency>
```

Comment:
It's establishing the version of `spring-cloud-stream-binder-rabbit` that child modules will use. Typically dependencies are managed in the parent or a `-dependencies` artifact, but since the project is too small, it doesn't warrant one.